src/player.cpp

```diff
@@ -4160,14 +4160,14 @@ void player::pause()
     search_surroundings();
 }
 
-void player::shout( std::string msg )
+void player::shout( std::string msg, const std::string &verbal_shout_id )
 {
-    int base = 10;
+    int base = 85;
```

Comment:
> A shout that would draw more hordes than NPCs sounds even less useful.

I disagree. A shout that could draw more hordes than NPCs give the player a risk/reward choice to make. There may not even be any hordes nearby, in which case, it can be very useful. On the other hand, a shout that reaches 5 squares beyond chat distance has no risk (it doesn't draw zombies, and you'd be in visual range of zombies at that distance), nor is it particularly useful since chatting could be used instead of shouting the majority of the time.

> a shout command to draw neutral NPCs near will have to be nearly useless, unless it somehow allowed drawing them from overmap or having them stop combat to respond to the shout.

Eh, it's useless only because the limited range is nearly equal to npc chat distance. As it was originally with the increased shout loudness, it was actually quite useful since it does allow calling NPCs from the overmap, a bit beyond local view distance.